dev/CommandBarFlyout/CommandBarFlyoutCommandBar.cpp

```diff
@@ -755,6 +763,27 @@ void CommandBarFlyoutCommandBar::EnsureFocusedPrimaryCommand()
     }
 }
 
+void CommandBarFlyoutCommandBar::MoreButtonClicked(const winrt::IInspectable& sender, const winrt::RoutedEventArgs& args)
+{
+    // If the button get's invoked, we switch between secondary commands visible and secondary commands not visible
+    m_secondaryCommandsExpectedToBeVisible = !m_secondaryCommandsExpectedToBeVisible;
+}
+
+void CommandBarFlyoutCommandBar::OverFlowPopupClosed(const winrt::IInspectable& sender, const winrt::IInspectable& args)
+{
+    // We expected our secondary commands to be visible, yet the flyout was closed.
```

Comment:
>expected [](start = 10, length = 8)

I think the secondary items menu is open by default on lower OS versions if my memory is correct